packages/powersync/lib/src/uuid.dart

```diff
@@ -2,4 +2,6 @@ import 'package:uuid/uuid.dart';
 import 'package:uuid/data.dart';
 import 'package:uuid/rng.dart';
 
-final uuid = Uuid(goptions: GlobalOptions(CryptoRNG()));
+import 'log_internal.dart';
+
+final uuid = kIsWeb ? Uuid() : Uuid(goptions: GlobalOptions(CryptoRNG()));
```

Comment:
Why not CryptoRNG on web?

Reply:
Initially I thought this caused compiler errors, but I was not able to reproduce this now - my initial suspicions must have been caused by something else. Reverted this change to use `CryptoRNG` for all platforms.